jekyll/_cci2/configuration-reference.md

```diff
@@ -198,10 +200,10 @@ It's possible to specify multi-line `command`:
       make test
 ```
 
-Note that default `shell` has `-e` option which makes command to:
+Note that default `shell` has `-e` option which causes commands to:
 > Exit immediately if a pipeline (which may consist of a single simple command), a subshell command enclosed in parentheses, or one of the commands executed as part of a command list enclosed by braces exits with a non-zero status.
 
-So if in previous example `mkdir` failed to create a dir and returned a non-zero status then command execution will be terminated and whole step will be marked as failed. If a desired behaviour is opposite, then you need to add `set +e` or override default `shell`:
+So if in the previous example `mkdir` failed to create a directory and returned a non-zero status, then command execution will be terminated, and the whole step will be marked as failed. If you desire the opposite behaviour, you need to add `set +e` in your `command` or override the default `shell` in your configuration map of `run`. For example:
```

Comment:
"...command execution *would* be terminated, and the whole step *would* be marked as failed."

Also "desire" over "want", @ndintenfass? So steamy, these docs. :)